pkg/dpm/plugin.go

```diff
@@ -107,6 +107,7 @@ func (dpi *DevicePlugin) register(kubeletEndpoint, resourceName string) error {
 	_, err = client.Register(context.Background(), reqt)
 	if err != nil {
 		glog.Errorf("Registration failed: %s", err)
+		glog.Errorf("Did you open the feature gate?")
```

Comment:
Something less assuming such as "Make sure that the DevicePlugins feature gate is enabled" would be nicer. :)

Reply:
done